Fig6_PyGMT_geopandas.py

```diff
@@ -0,0 +1,50 @@
+import geopandas as gpd
+import pygmt
+
+provider = "https://naciscdn.org/naturalearth"
+states = gpd.read_file(f"{provider}/50m/cultural/ne_50m_admin_1_states_provinces.zip")
+rivers = gpd.read_file(f"{provider}/50m/physical/ne_50m_rivers_lake_centerlines.zip")
+cities = gpd.read_file(f"{provider}/110m/cultural/ne_110m_populated_places_simple.zip")
+
+states = states[states["admin"] == "United States of America"].copy()
+states["area_sqkm"] = states.geometry.to_crs(epsg=6933).area / 10 ** 9
+rivers = rivers[rivers.intersects(states.union_all())].copy()
+cities = cities[cities["adm0name"] == "United States of America"].copy()

```

Comment:
1. Reorder the codes so that the first three lines focus on limiting the dataset to US. 
2. Remove the `.copy()` method, since they seem unnecessary in this case (i.e., no warnings).
3. Use `states.to_crs` rather than `states.geometry.to_crs`
```suggestion
states = states[states["admin"] == "United States of America"]
rivers = rivers[rivers.intersects(states.union_all())]
cities = cities[cities["adm0name"] == "United States of America"]

states["area_sqkm"] = states.to_crs(epsg=6933).area / 10 ** 9
```